### PR TITLE
update roadmap + added 4.0 dl links

### PR DESCRIPTION
**Pull request by Yakuza112**

### CODE REVIEW

File: _data/langs/de.yml [hunk 1]
```diff
@@ -187,26 +187,26 @@ roadmap:
...
     header: "I2P Android Geldbörse"
     paragraph: "Android Geldbörse ausschließlich nur für anonyme Transaktionen (kurz vor Fertigstellung)"
-    date: "Q4, 2017"
+    date: "Q1, 2018"
   core_wallet_release:
     header: "Core Wallet 3.0 Release Stage 1"
     paragraph: "Wallet UI Overhaul, VISP, Bloom Filters, Atomic Swaps Capability"
     date: "<font color='#08b4e0'>Veröffentlicht</font> 15. November 2017"
   core_wallet_release_2:
     header: "Core Wallet 3.0 Release Stage 2"
```
Comment (arunvinudss) [inline _data/langs/de.yml]: I think this should be "Core Wallet 4.0 Release Stage 1"

Comment (Yakuza112) [reply _data/langs/de.yml]: added

**Comment by justinvforvendetta** [comment]
merging after checks

**Comment by ekryski** [comment]
Awesome! This looks good to go now. :shipit:

**Pull request merged**